data/pages-content/sprints.md

```diff
@@ -63,10 +63,11 @@ As per our tradition, the conference organisers will provide the rooms and facil
 
 If you are planning to run a sprint at EuroPython 2023, please add it at the bottom of this page [creating a pull request to our website](https://github.com/EuroPython/website/blob/main/data/pages-content/sprints.md):
 
-1. Copy the template at the bottom of the sprint list.
-2. Fill the template with your details.
-3. Add your sprint at the end of the list.
-4. Update the room occupancy in the list above.
+1. Make sure your sprint is about an _open source_ project.
```

Comment:
Maybe also add a line about being mindful of how many people can they accommodate.. People who are not as aware of the project require way more hand-holding.